tests/testthat/test-transition_risk_profile.R

```diff
@@ -476,3 +476,107 @@ test_that("outputs `co2_footprint` at product level and `co2_avg` at company lev
   expect_true("co2_footprint" %in% names(unnest_product(output)))
   expect_true("co2_avg" %in% names(unnest_company(output)))
 })
+
+test_that("the output at product level has columns matching `postcode`, `address`, `min_headcount`, `max_headcount`, `emissions_profile_best_case`, `emissions_profile_worst_case`, `transition_risk_profile_best_case`, and `transition_risk_profile_worst_case`", {
+  withr::local_options(list(tiltIndicatorAfter.output_co2_footprint = TRUE))
```

Comment:
Does this test need this option? If so it may be time to make the option the default.

Reply:
Thanks for spotting! Removed!